helix-core/helix-core-1.0.2-SNAPSHOT.ivy

```diff
@@ -53,8 +53,8 @@ under the License.
         <artifact name="slf4j-log4j12" ext="jar"/>
     </dependency>
     <dependency org="org.apache.zookeeper" name="zookeeper" rev="3.4.13" conf="compile->compile(default);runtime->runtime(default);default->default"/>
-    <dependency org="org.codehaus.jackson" name="jackson-core-asl" rev="1.8.5" conf="compile->compile(default);runtime->runtime(default);default->default"/>
-    <dependency org="org.codehaus.jackson" name="jackson-mapper-asl" rev="1.8.5" conf="compile->compile(default);runtime->runtime(default);default->default"/>
+		<dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.11.0" force="true" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
```

Comment:
Just curious, what's the "master(*)" here?

Reply:
Ahh I copied from other ivy files.. I did not notice the difference between this one and others.. I will change it back to the old way.

Reply:
TFTR. Updated.